ironfish/src/consensus/verifier.test.ts

```diff
@@ -373,6 +373,166 @@ describe('Verifier', () => {
     })
   })
 
+  describe('Before postive block mine time activation', () => {
```

Comment:
What do you think about nesting these tests in the `verifyBlockHeaderContextual` block alongside the other tests? We have an existing test there, which I think makes a good place for these to go. We should also probably just remove the old test if it isn't relevant anymore: https://github.com/iron-fish/ironfish/blob/master/ironfish/src/consensus/verifier.test.ts#L521

Reply:
Sure, but I want to keep the current describe and it structures because its clear and easy to read when group the tests with before activation behavior and after activation behavior.